e2e/Pages/AggregateComponenet.ts

```diff
@@ -3,22 +3,73 @@ import { Locator, Page } from 'playwright';
 
 import DashboardPage from './DashboardPage';
 
+const expectedLayersNames = [
```

Comment:
It's good that this list is outside the logic. This list varies depending on country and hazard.